src/overlays/actors/ovl_Bg_Haka_Zou/z_bg_haka_zou.c

```diff
@@ -8,17 +8,43 @@
 
 #define FLAGS 0x00000010
 
+typedef enum {
+    /* 0x0 */ STA_GIANT_BIRD_STATUE,
+    /* 0x1 */ STA_BOMBABLE_SKULL_WALL,
+    /* 0x2 */ STA_BOMBABLE_RUBBLE,
+    /* 0x3 */ STA_UNKNOWN
+} ShadowTempleAssetsType;
+
 #define THIS ((BgHakaZou*)thisx)
 
 void BgHakaZou_Init(Actor* thisx, GlobalContext* globalCtx);
 void BgHakaZou_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void BgHakaZou_Update(Actor* thisx, GlobalContext* globalCtx);
+void BgHakaZou_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-extern UNK_TYPE D_06000A10;
-extern UNK_TYPE D_06005E30;
-extern UNK_TYPE D_06006F70;
+void BgHakaZou_Wait(BgHakaZou* this, GlobalContext* globalCtx);
+void func_80882BDC(BgHakaZou* this, GlobalContext* globalCtx);
+void func_80883000(BgHakaZou* this, GlobalContext* globalCtx);
+void func_80883104(BgHakaZou* this, GlobalContext* globalCtx);
+void func_80883144(BgHakaZou* this, GlobalContext* globalCtx);
+void func_80883254(BgHakaZou* this, GlobalContext* globalCtx);
+void func_80883328(BgHakaZou* this, GlobalContext* globalCtx);
+void func_808834D8(BgHakaZou* this, GlobalContext* globalCtx);
+void BgHakaZou_Do_Nothing(BgHakaZou* this, GlobalContext* globalCtx);
+
+extern Gfx D_06000A10[];
+extern ColHeader D_06005E30;
+extern ColHeader D_06006F70;
+extern ColHeader D_06000C2C;
+
+static ColliderCylinderInit sCylinderInit = {
+    { COLTYPE_UNK10, 0x00, 0x09, 0x00, 0x20, COLSHAPE_CYLINDER },
+    { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x00000008, 0x00, 0x00 }, 0x00, 0x01, 0x00 },
+    { 5, 60, 0, { 0, 0, 0 } },
+};
+
+static Vec3f sEffectVec = { 0.0f, 0.0f, 0.0f };
```

Comment:
This seems to be a typical zero vector, used for various purposes. We usually name it `sZeroVec` so I suggest using that here.
```suggestion
static Vec3f sZeroVec = { 0.0f, 0.0f, 0.0f };
```